StRoot/StFcsDbMaker/StFcsDb.cxx

```diff
@@ -515,9 +515,8 @@ int StFcsDb::getDetFromName(const std::string& detname){
     else if( detname=="ES") {return 1;}
     else if( detname=="HN") {return 2;}
     else if( detname=="HS") {return 3;}
-    else if( detname=="HN") {return 4;}
-    else if( detname=="PN") {return 5;}
-    else if( detname=="PS") {return 6;}
+    else if( detname=="PN") {return 4;}
+    else if( detname=="PS") {return 5;}
```

Comment:
If you call `getDetFromName("PN")` it would return 5, now it returns 4 for the same argument. Similarly, for "PS". Since the function is public, some code may rely on the specific returned values and can break after the change.

Reply:
"PN" should return 4, not 5. That was a part of the cut & paste & mis-edited bug...

Reply:
That's the problem with public interfaces when bugs cannot be fixed easily and sometimes have to be carried around... This is of course nowhere close to where it can be relevant.

Reply:
Sooo.... PN should keep returning 5 even though it is wrong? I guess David should have used local (unnamed namespace) const char* DET[6] for this.... since that is used original getName() function, and this is the partial reverse.